.ahub/tcchecker-tca/config.yaml

```diff
@@ -17,6 +17,7 @@ test:
       - /runtime/onert/test/ir
       - /runtime/onert/test/util
       - /tests/nnfw_api/src
+      - /runtime/libs/misc/test
```

Comment:
@hseok-oh I think it is the only line to be added, but please take a look. It is first time for me to edit this file.

Reply:
:+1: